src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartHeaderAdmin.cshtml

```diff
@@ -2,6 +2,11 @@
 @inject OrchardCore.ContentManagement.Display.IContentItemDisplayManager ContentItemDisplayManager
 @inject OrchardCore.DisplayManagement.ModelBinding.IUpdateModelAccessor UpdateModelAccessor
 
+@if (Model.ContainerContentItem == null)
```

Comment:
Why not done in a driver?

Reply:
yes. agreed! I made that change. Don't forget to approve if you have no additional issues with this PR.